static/app/views/replays/detail/errorList/errorHeaderCell.tsx

```diff
@@ -0,0 +1,85 @@
+import {ComponentProps, CSSProperties, forwardRef, ReactNode} from 'react';
+import styled from '@emotion/styled';
+
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconArrow, IconInfo} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useSortErrors from 'sentry/views/replays/detail/errorList/useSortErrors';
+
+type SortConfig = ReturnType<typeof useSortErrors>['sortConfig'];
+type Props = {
+  handleSort: ReturnType<typeof useSortErrors>['handleSort'];
+  index: number;
+  sortConfig: SortConfig;
+  style: CSSProperties;
+};
+
+const SizeInfoIcon = styled(IconInfo)`
+  display: block;
+`;
+
+const COLUMNS: {
+  field: SortConfig['by'];
+  label: string;
+  tooltipTitle?: ComponentProps<typeof Tooltip>['title'];
+}[] = [
+  {field: 'id', label: t('Event ID')},
+  {field: 'title', label: t('Title')},
+  {field: 'project', label: t('Issue')},
+  {field: 'timestamp', label: t('Timestamp')},
+];
+
+export const COLUMN_COUNT = COLUMNS.length;
+
+function CatchClicks({children}: {children: ReactNode}) {
+  return <div onClick={e => e.stopPropagation()}>{children}</div>;
+}
+
+const ErrorHeaderCell = forwardRef<HTMLButtonElement, Props>(
+  ({handleSort, index, sortConfig, style}: Props, ref) => {
+    const {field, label, tooltipTitle} = COLUMNS[index];
+    return (
+      <HeaderButton style={style} onClick={() => handleSort(field)} ref={ref}>
+        {label}
+        {tooltipTitle ? (
+          <Tooltip isHoverable title={<CatchClicks>{tooltipTitle}</CatchClicks>}>
+            <SizeInfoIcon size="xs" />
+          </Tooltip>
+        ) : null}
+        <IconArrow
+          color="gray300"
+          size="xs"
+          direction={sortConfig.by === field && !sortConfig.asc ? 'down' : 'up'}
+          style={{visibility: sortConfig.by === field ? 'visible' : 'hidden'}}
+        />
+      </HeaderButton>
+    );
+  }
+);
+
+const HeaderButton = styled('button')`
+  border: 0;
+  border-bottom: 1px solid ${p => p.theme.border};
+  background: ${p => p.theme.backgroundSecondary};
+  color: ${p => p.theme.subText};
+
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 600;
+  line-height: 16px;
+  text-align: unset;
+  text-transform: uppercase;
+  white-space: nowrap;
+
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: ${space(0.5)} ${space(1)} ${space(0.5)} ${space(1.5)};
+
+  svg {
+    margin-left: ${space(0.25)};
+  }
+`;
```

Comment:
Do our other tabs share this header button style (or something similar)?

Reply:
yes! there's loads of stuff copied from the network tab. First time it's being re-used is now.

Reply:
re-used aka copy+pasted

Reply:
What do you think would be easier to review:

(1) this as-is, with all the copy paste, then follow up with one or more PRs to de-dupe stuff.

or, (2) I can update this PR and extract all the common stuff between error-table and the network-table

or, (3) i can pull out the common stuff in a first PR, then rebase this and use all the new share-able bits